deepmd/utils/neighbor_stat.py

```diff
@@ -82,7 +82,12 @@ def get_stat(self,
                                                 self.place_holders['box']: np.array(data_set['box'])[kk].reshape([-1, 9]),
                                                 self.place_holders['default_mesh']: np.array(data.default_mesh[ii]),
                                             })
-                    dt = np.min(dt)
+                    if dt.size != 0:
+                        dt = np.min(dt)              
+                    else:
+                        dt = self.rcut
+                        log.warn("Atoms with no neighbors found in %s. Please make sure it's what you expected."%jj)
```

Comment:
https://docs.python.org/3/library/logging.html#logging.Logger.warning

> As `warn` is deprecated, please do not use it - use `warning` instead.

Reply:
Sorry! I have modified log.warn to log.warning